Tasks/Mardoyan/Task3/Task3_1.py

```diff
@@ -0,0 +1,11 @@
+print('Hello! I can help you count the vowels in your string.')
+x = input('Input you string(on English)\n')
+vowels = 0
+for i in x:
+    letter = i.lower()
+    if letter == "a" or letter == "i" or\
+       letter == "u" or letter == "e" or\
+       letter == "y" or letter == "o":
+       vowels += 1
```

Comment:
would be better to have all letter stored in a list and perform check with the "in" operator